src/main/java/frc/robot/RobotContainer.java

```diff
@@ -203,8 +213,8 @@ private void configureButtonBindings() {
     m_driverController.leftTrigger().whileTrue(new IntakeNote());
     m_driverController.rightTrigger().whileTrue(new Puke()).onFalse(new ResetWheels());
     m_driverController.leftBumper().onTrue(armUp);
-    m_driverController.rightBumper().whileTrue(new PivotDown(0.5, -0.1));
-    m_driverController.y().whileTrue(new AlignToTrap());
+    m_driverController.rightBumper().whileTrue(new PivotDown(0.75, -0.1));
+    m_driverController.y().onTrue(feederShot); // m_driverController.y().whileTrue(AlignToTrap());
```

Comment:
```suggestion
    m_driverController.y().onTrue(feederShot); // m_driverController.y().whileTrue(new AlignToTrap());
```

Why'd you remove new keyword?